Sources/Prometheus/MetricTypes/Histogram.swift

```diff
@@ -41,7 +41,7 @@ public struct Buckets: ExpressibleByArrayLiteral {
         assert(count >= 1, "Bucket.linear needs a count larger than 1")
         var arr = [Double]()
```

Comment:
random drive-by but we could make all this code

```
let arr = (1...count).map { Double($0) * Double(start) }
```

which would make it slightly more efficient and just one line.

Feel free to ignore.

Reply:
Sure, why not.